framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -405,18 +405,23 @@ player_result_t MediaPlayerImpl::setVolume(int vol)
 	return ret;
 }
 
-void MediaPlayerImpl::setVolumePlayer(int vol, player_result_t &ret)
+void MediaPlayerImpl::setVolumePlayer(uint8_t vol, player_result_t &ret)
```

Comment:
same as above regarding &ret

Reply:
I didn't change the ret as a pointer since junmin suggested me to do it in another PR.
I agree with junmin, how do you think?

Reply:
Sure, why not?